src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from '@emotion/styled';
+import { ReactNode } from 'react';
+import { base } from '../../../tokens';
+
+// color mapping key:value pair for strings to bg color
+const SEVERITY_TO_BG_COLOR_MAPPING = {
+  success: base.color.green50,
+  error: base.color.red50,
+  warning: base.color.yellow50,
+  info: base.color.sky50,
+}
```

Comment:
i think we can do as const on this 🤔 
```suggestion
const SEVERITY_TO_BG_COLOR_MAPPING = {
  success: base.color.green50,
  error: base.color.red50,
  warning: base.color.yellow50,
  info: base.color.sky50,
} as const
```